website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Enhancement: New features and UI changes to dbt Explorer"
+description: "November 2023: New features and UI changes to dbt Explorer, including a new filter panel, improved lineage graph, and detailed resource information."
+sidebar_label: "Enhancement: New features and UI changes to dbt Explorer"
+sidebar_position: 08
+tags: [Nov-2023]
+---
```

Comment:
i think the date property was missed! might be good to post in the dbt cloud community channel